spec/controllers/queue_items_controller_spec.rb

```diff
@@ -83,6 +83,12 @@
         delete :destroy, id: item1.id
         expect(response).to redirect_to my_queue_path
       end
+      
+      it "normalizes remaining queue items" do
+        item2 = Fabricate(:queue_item, user: george, position: 2)
+        delete :destroy, id: item1.id
+        expect(QueueItem.first.position).to eq(1)
```

Comment:
you know i had it written that way originally but changed it when i watched the videos because i thought maybe i was wrong.